inst/components/scss/toolbar.scss

```
@@ -1,5 +1,11 @@
 /* Toolbar */
 .bslib-toolbar {
+  --_divider-height: var(--bslib-toolbar-divider-height, 1lh);
+  --_divider-width: var(--bslib-toolbar-divider-width, 2px);
+  --_divider-gap: var(--bslib-toolbar-divider-gap, 1rem);
```

I think we should go ahead and follow the same pattern for the divider color

```suggestion
  --_divider-gap: var(--bslib-toolbar-divider-gap, 1rem);
  --_divider-color: var(--bslib-toolbar-divider-color, var(--bs-border-color-translucent, rgba(40,70,94,0.1)));
```

I'll let you decide if you think we should let users customize this in `toolbar_divider()`.

---

Works for me-- I think we should leave it as a CSS-only change to keep the args a bit slimmer. If someone wants to change it on a toolbar-by-toolbar basis in the code, they can pass it in like:
```
        toolbar(
          style = "--_divider-color: red;",
          toolbar_input_button(
            id = "btn1f",
            label = "Home",
            icon = icon("home")
          ),
          toolbar_divider(), # Default divider
```